java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -221,4 +221,39 @@ public interface SetBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> sunionstore(String destination, String[] keys);
+
+    /**
+     * Returns a random element from the set value stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/srandmember/">redis.io</a> for details.
+     * @param key The key from which to retrieve the set member.
+     * @return A random element from the set, or <code>null</code> if <code>key</code> does not exist.
+     * @example
+     *     <pre>{@code
+     * client.sadd("test", new String[] {"one"}).get();
+     * String response = client.srandmember("test").get();
+     * assertEquals("one", response);
+     * }</pre>
+     */
+    CompletableFuture<String> srandmember(String key);
+
+    /**
+     * Returns an array of distinct random elements from the set value stored at <code>key</code> if a
+     * positive <code>count</code> was given, or an array of random elements allowing duplicates if a
+     * negative <code>count</code> value was given. In the negative case, the number of elements is
+     * the absolute value of <code>count</code>.
```

Comment:
Move this to `@param count` section. Please see `hrandmember` and `zrandmember` for examples.